test/e2e/subaccount_service_broker_v1alpha1_test.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Sorry should have been more clear 
I meant removing this e2e tag just temporarily for you locally to see compiler errors. 
I needs to be there for the for the code to properly work I think.